lib/query-builders/admin.js

```diff
@@ -22,6 +22,7 @@ module.exports = {
   outstanding: (queryBuilder, profile) => {
     queryBuilder
       .where(buildEstablishmentQuery(profile))
+      .whereJsonSupersetOf('data', { initiatedByAsru: false })
```

Comment:
is this going to potentially hide old tasks that do not have the `initiatedByAsru` prop?

Reply:
Yes, it absolutely is.

I'll see what impact that has.

Reply:
do you want to hold off merging this?